src/components/font_size_editor/font_size_editor.ts

```diff
@@ -31,6 +31,14 @@ css/* scss */ `
     input::-webkit-inner-spin-button {
       -webkit-appearance: none;
     }
+
```

Comment:
Probably already there in master, but colorPicker dropdown is lower than all the other menus in the topbar